pygeoapi/provider/mvt_postgres.py

```diff
@@ -0,0 +1,246 @@
+# =================================================================
+#
+# Authors:
+#
+# Copyright (c)
```

Comment:
Please add copyright.  Example in https://github.com/geopython/pygeoapi/blob/master/setup.py#L5